### PR TITLE
Rename assetPath to revisionedPath and readd assetPath with different behaviour

### PR DESCRIPTION
**Pull request by jakub300**
I wasn't sure what kind of error we should do on WP, I felt like throwing may be too radical, let me know what do you think.

### CODE REVIEW

**Comment by jakub300** [comment]
New version should reflect @thymikee comment from #119.

File: generators/app/templates/gulp/templatesFunctions.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+var pathModule = require('path');
+
+module.exports = function(data) {
+  data = data || {};
+  var functions = [
+    {
+      name: "revisionedPath",
```
Comment (thymikee) [inline generators/app/templates/gulp/templatesFunctions.js]: Can we use single quotes here?

File: generators/app/templates/gulp/templatesFunctions.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+          }
+          return pathModule.join(pathModule.dirname(fullPath),
+            data.manifest[path]);
+        } else {
+          return path;
+        }
+      }
+    },
+    {
+      name: 'assetPath',
```
Comment (thymikee) [inline generators/app/templates/gulp/templatesFunctions.js]: I'd rename it to `assetsPath`. Plural form is already used in a variable `ASSETS_PATH`, also the name of the folder is `assets`. Just for consistency here

Comment (jakub300) [inline generators/app/templates/gulp/templatesFunctions.js]: Well, function returns path to single asset and variable describes directory with many assets :D

Comment (thymikee) [inline generators/app/templates/gulp/templatesFunctions.js]: Damn, I was overcomplicating this one. Leave this as is 👍

**Pull request merged**